package.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "gatsby-plugin-umami",
+    "name": "gatsby-plugin-umami-ng",
     "description": "Gatsby plugin to add Umami to a Gatsby site",
     "version": "0.1.3",
-    "author": "Florian Kapfenberger <florian@kapfenberger.me>",
+    "author": "Henri Chilla <henri@chilla.dev>",
```

Comment:
thanks for your contribution, but pls revert this change so the npm package can still be published by me

I would be happy if we would introduce this: https://github.com/all-contributors/all-contributors 
to honor your and other future contributions. Feel free to open another PR

Reply:
Yes, that was my mistake. I'll revert it.